### ISSUE TITLE
Designed a better UX for meeting participants to indicate they’re ready

### ISSUE THREAD

**Issue by enriquesanchez** [ParabolInc/parabol#5420]
## Issue - Enhancement

From #4951:

> During a Retrospective today, I kept having to ask the participants if they were ready to move to the next phase. According to them, they click a Ready button when they are Ready. I as a facilitator have no way of seeing when they are. If would be super helpful if I can get an indication or notification when everyone is ready to move on.

> The ready button is really confusing, people don't know if they have hit ready or not. We just want to see the ready button get fixed as soon as possible.

>I would like a better way of knowing who has and hasn't hit "next" to make it easier for me to know when to move on to the next phase

We need a solution that improves clarity of the ready status for both participants and facilitators. 

- How can we can make the Ready button more prominent and clear?
- How can we make it easier for facilitators to know when its time to move on to the next phase?
- What kind of solution can we design that takes into account user presence and async meetings

Some initial exploratory work can be found [here](https://www.notion.so/parabol/Participation-status-IA-82eab208a48d4884be613f4d78907bea) 🔒 and [here](https://www.notion.so/parabol/User-presence-activity-status-5599fa99e8e643ff9c1f0bb53e000f39) 🔒.
 

### Acceptance Criteria
- Design and end-to-end concept of a solution that improves clarity of the ready status for both participants and facilitators
- Iterate as necessary
- Create implementation issue

### Estimated effort
- 48 hrs.

**Comment by jordanh** [comment]
This should be a design batting practice. Will adjust now

**Comment by jordanh** [comment]
[A user](https://app.hubspot.com/contacts/3888472/contact/23426101) writes:

> Ps: a complain I ear often is that the “ready” button is confusing and hard to distinguish between checked and non checked, especially if more people do it closely

**Comment by ackernaut** [comment]
A user writes:

> Make it more clear that I've selected the "Ready" green check after each activity (I couldn't read the indicator to tell that I had checked it or not)

**Comment by garethaledavies** [comment]
A [user](https://www.g2.com/products/parabol/reviews/parabol-review-7357261) writes:

> Seems very nit picky but in all of my teams people tend to struggle with the 'Next Button'. It's super neat how it slowly fills up the circle until everyone clicked 'Next', but the issue is that 1) most people forget to click the Next button, 2) don't know whether they have already clicked it (resulting in clicking twice which marks them as "not ready") and 3) don't immediately get that the green circle shows the overall "ready state" of the team. So making this button a bit bigger, maybe have it buzz or something if there is no more activity on the screen would be super helpful.

**Comment by enriquesanchez** [comment]
[A customer writes](https://app.hubspot.com/live-messages/3888472/inbox/3627165845#live-chat):

>I get a check mark for "Ready" but only half of the circle is complete.  Do I need to do anything else to save this?

**Comment by enriquesanchez** [comment]
[A customer](https://app.hubspot.com/live-messages/3888472/inbox/3642812016#email) writes 🔒:

>Our team loves using your product for retros. However, people can never remember or find the buttons to indicate that they are done with a given step (e.g., voting or adding comments). 
Please make the button bigger, move it to the top, or otherwise make it more discoverable.

**Comment by drewhousman1010** [comment]
A [user ](https://app.hubspot.com/contacts/3888472/contact/41958351) 🔒 said on a call:

> People don't hit ready

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/87718201) 🔒  said on a call: 

> People don't press it. Can you make a better prompt to encourage people to click? People don't know what it does, or don't know that it alerts the the facilitator

**Comment by jordanh** [comment]
This one went stale. Iceboxing. Please re-open should we hear this feedback again
